.drone.yml

```diff
@@ -33,6 +33,16 @@ steps:
       GOSUMDB: sum.golang.org
       TAGS: bindata sqlite sqlite_unlock_notify
 
+  - name: lint-backend-gogit
+    pull: always
+    image: golang:1.15
+    commands:
+      - make lint-backend
+    environment:
+      GOPROXY: https://goproxy.cn # proxy.golang.org is blocked in China, this proxy is not
+      GOSUMDB: sum.golang.org
+      TAGS: bindata gogit sqlite sqlite_unlock_notify
+
```

Comment:
Sorry I'm late here but is it really necessary to run the linter twice here? Do `TAGS` really affect the linting result? I thought `TAGS` only affect the build.

Reply:
yes

Reply:
I guess it's beyond me to understand how TAGS can affect linting (which should just be a static analysis of files). Care to explain? Does the go linter actually do runtime analysis as well?

Reply:
golangci-lint has a `--build-tags` to support lint different tags.

Reply:
Ok I guess I need to study `golangci-lint` more. I was under the impression that linters in general just do static analysis of file contents, but I guess this linter does more than that.

Reply:
Thought I generally wonder if it's really necessary to do this double linting every single CI run. It takes like 3 minutes which is already pretty slow.

Reply:
The issue is that the go-git variants require linting too.

Reply:
Can't we just pass a static list of files to the linters? I think it should be possible.